Cargo.toml

```diff
@@ -13,3 +13,4 @@ categories = ["embedded", "no-std", "hardware-support", "os"]
 
 [dependencies]
 tock-registers = "0.8"
+spinlock = { path = "../../crates/spinlock" }
```

Comment:
emmmmmmm

Reply:
emmm, sorry.

Reply:
most importantly, arm_gicv2 should not hold spinlock or dep on spinlock

Reply:
arm_gicv2 should not hold spinlock or dep on spinlock